portfolio/src/main/webapp/SlideController.js

```diff
@@ -0,0 +1,23 @@
+import SlideHandler from '/SlideHandler.js';
+
+/** The controller handling all slideshows. */
+export default class SlideController {
```

Comment:
Here and elsewhere don't use default exports.

https://google.github.io/styleguide/jsguide.html#es-module-exports

Reply:
Thanks! I missed that when I was reading style guide (sorry)